[Samsung mobile] Update EOL info for many device (#9907)
endoflife-date/endoflife.date: products/samsung-mobile.md

    releaseLabel: "Galaxy A16 5G"
    releaseDate: 2024-10-25
    eoas: 2030-10-25 # "6 generations of OS updates" (https://news.samsung.com/uk/samsung-introduces-the-galaxy-a16-series-featuring-two-new-devices)

>     eol: 2030-10-31      # "6 years of security updates"

Reviewer: ```suggestion
    eol: 2030-10-25      # "6 years of security updates"
```

    releaseLabel: "Galaxy XCover7"
    releaseDate: 2024-01-10
    eoas: 2030-01-10 #  "ANDROID 21 supported"
    eol: 2030-01-10      # "ANDROID 21 supported"
    link: https://doc.samsungmobile.com/sm-g556b/eux/doc.html

  - releaseCycle: "galaxy-a25-5g"
    releaseLabel: "Galaxy A25 5G"
    releaseDate: 2023-12-11
    eoas: 2027-12-11 # "4 generations of OS upgrades" (https://news.samsung.com/in/samsung-galaxy-a25-5g-galaxy-a15-5g-with-awesome-camera-and-new-editing-features-launched-in-india)

>     eol: 2032-12-12      # "5 years of security updates"

Reviewer: ```suggestion
    eol: 2028-12-11      # "5 years of security updates"
```

    releaseLabel: "Galaxy A05s"
    releaseDate: 2023-10-18 # https://www.gsmarena.com/samsung_galaxy_a05s-12584.php
    eoas: 2025-10-18 # 2 android updates - https://rmupdate.com/galaxy-a05-software-update-policy/
    eol: 2027-10-18     # 4 years of security updates - https://rmupdate.com/galaxy-a05-software-update-policy/
    link: https://doc.samsungmobile.com/SM-A057F/FWD/doc.html

  - releaseCycle: "galaxy-a05"
    releaseLabel: "Galaxy A05"
    releaseDate: 2023-10-15 # https://www.gsmarena.com/samsung_galaxy_a05-12583.php
    eoas: 2025-10-15 # 2 android updates - https://rmupdate.com/galaxy-a05-software-update-policy/

>     eol: 2027-10-18      # 4 years of security updates - https://rmupdate.com/galaxy-a05-software-update-policy/

Reviewer: ```suggestion
    eol: 2027-10-15      # 4 years of security updates - https://rmupdate.com/galaxy-a05-software-update-policy/
```

    releaseLabel: "Galaxy A14 5G"
    releaseDate: 2023-01-05 # https://news.samsung.com/sg/introducing-the-samsung-galaxy-a14-5g-delivering-an-awesome-experience-for-all
    eoas: 2025-01-05 # 2 android upgrades - https://news.samsung.com/sg/introducing-the-samsung-galaxy-a14-5g-delivering-an-awesome-experience-for-all
    eol: 2027-01-05      # 4 years of security updates - https://news.samsung.com/sg/introducing-the-samsung-galaxy-a14-5g-delivering-an-awesome-experience-for-all
    link: https://doc.samsungmobile.com/SM-A146B/INS/doc.html

  - releaseCycle: "galaxy-m04"
    releaseLabel: "Galaxy M04"
    releaseDate: 2022-12-16
    eoas: 2024-12-16

>     eol: 2025-12-16

Reviewer: ```suggestion
    eol: false
```

Still listed on https://security.samsungmobile.com/workScope.smsb.

    releaseLabel: "Galaxy A23 5G"
    releaseDate: 2022-09-02
    eoas: 2025-09-02

>     eol: 2026-03-26

Reviewer: ```suggestion
    eol: false
```

Still listed on https://security.samsungmobile.com/workScope.smsb.

    releaseLabel: "Galaxy M53 5G"
    releaseDate: 2022-04-22 # https://news.samsung.com/in/samsung-launches-galaxy-m53-5g-with-segment-best-108-mp-quad-camera-segment-only-auto-data-switching-segment-leading-samoled-display-in-india
    eoas: 2024-04-22 # 2 android updates - https://androidspotlight.com/software-update-policy-for-every-samsung-device/

>     eol: 2025-04-22 # 3 years of security updates - https://androidspotlight.com/software-update-policy-for-every-samsung-device/

Reviewer: ```suggestion
    eol: false # 3 years of security updates - https://androidspotlight.com/software-update-policy-for-every-samsung-device/
```

Still listed on https://security.samsungmobile.com/workScope.smsb.

    releaseLabel: "Galaxy A13"
    releaseDate: 2022-03-23
    eoas: 2025-03-23

>     eol: 2026-03-23

Reviewer: ```suggestion
    eol: false
```

https://security.samsungmobile.com/workScope.smsb

    releaseLabel: "Galaxy XCover5"
    releaseDate: 2021-03-12
    eoas: 2024-03-12

>     eol: 2025-03-12

Reviewer: ```suggestion
    eol: false
```

https://security.samsungmobile.com/workScope.smsb
